core/server/common/src/main/java/alluxio/cli/validation/ValidationTask.java

```diff
@@ -25,6 +25,13 @@
    */
   List<Option> getOptionList();
 
+  /**
+   * Checks whether the validation task is applicable.
+   *
+   * @return true if this validation task should be skipped, false otherwise
+   */
+  boolean shouldSkip();
```

Comment:
optional: we could have `validate` return a `ValidationResult` which could be `PASS`, `FAIL`, or `SKIPPED`.

Reply:
+1

Reply:
Can we do that in another PR?

Reply:
another PR is fine with me